Cloud April Release Notes (#16699)
cockroachdb/docs: _includes/releases/cloud/2023-04-10.md

## April 10, 2023

<h3> General changes </h3>

> - Contract customers can now [create {{ site.data.products.dedicated }} advanced clusters](../cockroachcloud/create-your-cluster.html#step-1-start-the-cluster-creation-process), which have all the features of {{ site.data.products.dedicated }} standard clusters, plus security features needed for [PCI compliance](../cockroachcloud/pci-dss.html). To upgrade your organization to a contract, [contact us](https://cockroachlabs.com/contact-sales).

Reviewer: ```suggestion
- Contract customers can now [create {{ site.data.products.dedicated }} advanced clusters](../cockroachcloud/create-your-cluster.html#step-1-start-the-cluster-creation-process), which have all the features of {{ site.data.products.dedicated }} standard clusters, plus security features needed for [PCI DSS compliance](../cockroachcloud/pci-dss.html). To upgrade your organization to a contract, [contact us](https://cockroachlabs.com/contact-sales).
```
It's the whole thing. :)

Author: Always? There are a few other places I can track down in the docs (and in the UI I think) where it's just PCI

Reviewer: "Payment Card Industry Data Security Standard" is what it stands for, so you can see that just "PCI" is under-specified. See https://listings.pcisecuritystandards.org/documents/PCI_DSS-QRG-v3_2_1.pdf. This is a different PCI than "Peripheral Component Interconnect", which is a hardware bus / standard.

> - Contract customers can now [create {{ site.data.products.dedicated }} advanced clusters](../cockroachcloud/create-your-cluster.html#step-1-start-the-cluster-creation-process), which have all the features of {{ site.data.products.dedicated }} standard clusters, plus security features needed for [PCI compliance](../cockroachcloud/pci-dss.html). To upgrade your organization to a contract, [contact us](https://cockroachlabs.com/contact-sales).

Reviewer 2: I tested that and it won't fix the problem. The problem seems to be due to the fact that the second bullet below starts with `{{ site.data.products.dedicated }}`. Definitely a bug with Vale. I'll log a ticket.

> - {{ site.data.products.dedicated }} [third party metrics integrations](../{{site.versions["stable"]}}/third-party-monitoring-tools.html) (Datadog and Amazon CloudWatch) will now [export the following metrics](../cockroachcloud/export-metrics.html) to third party targets:

Reviewer: This is a substantial edit and I may be off base, but I'm working on other docs related to metrics export right now and it seems like this might be the intent? I've updated all links to Dedicated pages rather than the self-hosted docs since these are the Cloud release notes. WDYT? Feel free to edit my edit. :)
```suggestion
- {{ site.data.products.dedicated }} clusters now [export the following metrics](../cockroachcloud/export-metrics.html#the-metricexport-endpoint) the following metrics to third-party monitoring tools such as [Datadog](../cockroachcloud/export-metrics.html?filters=datadog-metrics-export) and [Amazon CloudWatch](../cockroachcloud/export-metrics.html?filters=aws-metrics-export).:
```

  - `storage_l0_sublevels`
  - `storage_l0_num_files`
  - `security_certificate_expiration_ca`
  - `sql_mem_root_current`
  - `sys_host_disk_iopsinprogress`
  
<h3> Cloud API changes </h3>

> - [Cloud API](../cockroachcloud/cloud-api.html) users can now upgrade their cluster's [CockroachDB version](index.html), as well as roll back or manually finalize pending upgrades.

Reviewer: ```suggestion
- You can now upgrade, roll back, or manually finalize a pending of a cluster using [Cloud API](../cockroachcloud/cloud-api.html).
```
I'm not sure if we have a rule about second vs third person in release notes that is different from the preference for second person throughout the narrative docs. WDYT? This is a bit more action-oriented.

> - The [Cloud API endpoints](../cockroachcloud/cloud-api.html) for managing metric export configurations have been replaced with endpoints specific to the third party tool being used. For example, `EnableMetricExport` is now replaced by `EnableDatadogMetricExport` and `EnableCloudWatchMetricExport`.

Reviewer: Are the old points deprecated for a period of time or are they no longer available in the API? It seems like this could require code changes at the customer side.
```suggestion
- The [Cloud API endpoints](../cockroachcloud/cloud-api.html) for configuring metric export have been replaced with specific endpoints for each supported third-party integration. For example, `EnableMetricExport` is now replaced by `EnableDatadogMetricExport` and `EnableCloudWatchMetricExport`.
```

<h3> Security updates </h3>

> - [Admins](../cockroachcloud/console-access-management.html#console-admin) of organizations that have [enabled SSO](../cockroachcloud/cloud-org-sso.html) can now reset the passwords of other users in their organization.

Reviewer: PTAL -- there are no passwords to reset if the user is logging in using an SSO login method with basic (unconfigured) SSO. Their previous password is deleted. Only organizations who enable Cloud Organization SSO (the full one) and still have password auth enabled will have passwords stored for some users and not others.
```suggestion
- [Admins](../cockroachcloud/console-access-management.html#console-admin) of organizations that have [enabled Cloud Organization SSO](../cockroachcloud/cloud-org-sso.html) can now reset the passwords of other users in their organization who authenticate using passwords rather than an SSO authentication method.
```

<h3> Bug fixes </h3>

> - Fixed a bug where certain pages in the console would give a `404` error when navigating to the **Clusters** page.

Reviewer: ```suggestion
- Fixed a bug where  a `404` error would display when navigating from certain pages to the **Clusters** page.
```
